Marlin/src/inc/Conditionals_post.h

```diff
@@ -330,6 +330,13 @@
     #else
       #define PSU_NAME "ATX"      // ATX style
     #endif
+    #if DISABLED(AUTO_POWER_CONTROL)
```

Comment:
This is wrapped in `#ifndef PSU_NAME` so if you have a `PSU_NAME` defined but don't explicitly set `PSU_POWERUP_DELAY` you get a build error:
`Marlin\src\gcode\control\M80_M81.cpp:75:13: error: 'PSU_POWERUP_DELAY' was not declared in this scope`

Maybe the `#ifndef PSU_NAME` and `#if ENABLED(PSU_CONTROL)` right above this need to be tweaked?